client/src/services/index.js

```diff
@@ -24,7 +24,15 @@ export const createPaymentIntent = ({ cart }) => async (dispatch) => {
 
   try {
     // fetch to post payment internet
-    console.log(cart);
+    const result = await axios({
+      url: "/api/payment",
+      method: "POST",
+      body: JSON.stringify({ cart }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
```

Comment:
we should decide at some point on whether to use axios in here or using the requestHandler file in the utils folder.

Reply:
How do those handlers differ from each other?

I appreciate wrapping axios so we can abstract away the AJAX library. If we're separating functionality by request method, shouldn't those methods be destructured into the `config`? I'm also not fond of returning false by default for the error, unless we use the validateStatus option to prevent axios from throwing on 4xx and 5xx responses.
```js
{
  ...config,
  method: "{request method}",
  validateStatus: (status) => true
}
```

Reply:
To keep things RESTful, we want to handle status codes differently depending on the necessary changes in FE state to resolve the error (especially in 4xx) or let the user know that the request was completed with a server error occurring and to attempt the request again